packages/formation/sass/_shame.scss

```diff
@@ -3,77 +3,58 @@
 // is for this file to be empty, but some times you just have to throw that
 // hack in there to get it working.
 
-// message while loading react app
-.loading-message {
-  padding: 2em 0;
-  text-align: center;
-}
-
 body {
+  // used a lot in vets-website
   .row {
     max-width: 62.5em;
   }
 
   // We have no container element, so elements that are full width have to redefine the container layout every time
   // if they want to line up.
+  // unused in content-library
+  // Used in vets-website, used in src/platform/monitoring/DowntimeNotification/components/DowntimeApproaching.jsx 
   .row-padded {
     max-width: 100rem;
     padding: 0 1rem;
     margin: 0 auto;
   }
 }
 
+// unused in content-library
+// used in vets-website, used in src/applications/edu-benefits/0993/containers/FormPage.jsx
+// and src/applications/appeals/testing/components/ContactInformation.jsx
 .form-panel {
   @include media($medium-screen) {
     max-width: $usa-form-width;
   }
 }
 
-.main-form,
+// unused in content-library
+// used in vets-website, used in src/platform/forms-system/src/js/containers/FormApp.jsx 
+// and src/applications/simple-forms/mock-alternate-header-21-0845/components/CustomFormApp.jsx
 .progress-box {
   button {
     width: 100%;
   }
 }
 
-.button-icon {
-  font-size: 1.7rem;
-}
-
 .progress-box {
   border: 1px solid $color-gray-lightest;
   margin: 1.5rem 0;
   padding: 1rem 2rem;
 }
 
-.form-section {
-  display: none;
-}
-
-#content .section {
-  fieldset {
-    h4 {
-      padding: 0;
-    }
-  }
-}
 
-.edit-checkbox {
```

Comment:
I see one usage of `edit-checkbox` in vets-website [here](https://github.com/department-of-veterans-affairs/vets-website/blob/804fc87411ee25b96f1424ac26f1473fb2624daf/src/applications/facility-locator/sass/facility-locator.scss#L363-L365), but it's only styling `label:visited`. Do we know if this instance relies on these styles for the states below?

Reply:
I can't find any actual uses of `edit-checkbox` in the markup, so we should be safe. [Double checked](https://dsva.slack.com/archives/C0FQSS30V/p1700082344533909?thread_ts=1700080546.822219&cid=C0FQSS30V) with someone working on the facilities team.